Bio/motifs/jaspar/db.py

```diff
@@ -69,12 +94,18 @@
 import sys
 
 if sys.version_info < (3,):
+    """
```

Comment:
Rather than triple-quoted strings (used for docstrings), for comments just use the ``#`` syntax, e.g.

```
# Python 2
```